lib/media_items/index.js

```diff
@@ -27,27 +28,27 @@ class MediaItems {
   }
 
   async uploadMultiple(albumId, files, directoryPath) {
-    if(files.length > 50) {
-      throw new Error(`:batchCreate restricts maximum media items to 50. 
-        Make multiple sequential calls to uploadMultiple to avoid this limitation. 
-        See: https://developers.google.com/photos/library/guides/upload-media`)
-    }
     const url = `${constants.BASE_PATH}/:batchCreate`
-    const newMediaItems = await Promise.all(files.map(async (file) => {
-      let token = await this.transport.upload(file.name, path.join(directoryPath, file.name))
-      return {
-        description: file.description || "",
-        simpleMediaItem: {
-          uploadToken: token
+    const batchedFiles = chunk(files, 50)
+    for(const batch of batchedFiles) {
+      await new Promise(resolve => setTimeout(resolve, 10000)); // HACK: google is limiting requests per minute. this pauses to get around that.
```

Comment:
could we do this at the end? this means we wait `10000ms` before starting to upload?

Reply:
That sounds good to me